aiohttp/web.py

```diff
@@ -647,6 +662,90 @@ def write_eof(self):
         yield from super().write_eof()
 
 
+class WebSocketResponse(StreamResponse):
```

Comment:
i think we also need `can_start_websocket()` method somewhere, i'd prefer to have it on WebSocketResponse.